package.json

```diff
@@ -107,7 +107,7 @@
     "react-native-pdf": "^6.6.2",
     "react-native-performance": "^4.0.0",
     "react-native-permissions": "^3.0.1",
-    "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#84ee97dec11c2e65609511eb5a757d61bbeeab79",
+    "react-native-picker-select": "git+https://github.com/cubuspl42/react-native-picker-select.git#039abcb9cd6e444538537e3da5c1af1ad970ab8d",
```

Comment:
At the end of the PR review:
* Verify that the SHA1 is up-to-date
* **Replace the prefix** with `Expansify`

Let's keep this thread open.

Reply:
At the moment of writing, in the newest revision, the hash is up-to-date. We'll still need to switch to the `Expensify` prefix once the other PR is merged.

Reply:
Up-to date again, prefix still to be updated

Reply:
SHA1 & prefix updated